### PR TITLE
Issue #2653492 by yanniboi: Added features config exports for Individ…

### PR DESCRIPTION
**Pull request by yanniboi**
…ual and Organisation users.

This is tracked on d.o here: https://www.drupal.org/node/2653492

### CODE REVIEW

**Comment by andrewbelcher** [comment]
Is there any reason why we have the indiv/org submodules? It seems a little overkill having 3 modules to do this?

`modules/simple_crm/decoupled_auth_simple_crm.info.yml` appears to be missing a trailing whitespace, as are the indiv/org `.install` files.

Looking at it installed, all looks good with the exception of user 1/adminstrator getting all profiles.

**Comment by andrewbelcher** [comment]
@yanniboi @jamieaa64 I think it would be good to tidy up the naming:

## Module

Would `decoupled_auth_simple_crm` or `decoupled_auth_contacts` be better? I think everything in here so far could be easily depended on by Contacts when we do our more advanced stuff, so I'm inclined to go with the later? What do we think?

Also, is there any reason for the config to be separate to the module? I suppose it means in the future would could have Contacts depend on the config module but not the UI module...

## Profiles

I would be inclined to go with either `crm_indiv/org` or `contacts_indiv/org` depending on the above..?

## Fields

We don't need `field_` prefixes as they are what's provided by the UI. I'm wondering if we want to go with `contacts_indiv_*` and `contacts_org_*` (similar to OpenCRM in D7)? Or if we stick with `decoupled_auth_simple_crm` for the module name, perhaps `crm_indiv_*` and `crm_org_*`

Also, could we get photo for individuals as we have logo for organisations?

**Comment by yanniboi** [comment]
Refactored machine names as follows:

individual ->crm_indiv

organisation->crm_org

field_crm->crm

File: config/schema/decoupled_auth.views.schema.yml [hunk 1]
```diff
@@ -6,3 +6,7 @@ views.field.user_decoupled:
 views.filter.user_decoupled:
   type: views_filter
   label: 'Decoupled'
+
+views.filter_value.user_decoupled:
+  type: string
+  label: 'Has web account?'
```
Comment (andrewbelcher) [inline config/schema/decoupled_auth.views.schema.yml]: Still missing the new line at the end of this file.

File: travis-ci/travis-before-script.sh [hunk 2]
```diff
@@ -6,11 +6,28 @@ set -e $DRUPAL_TI_DEBUG
 # Note: This function is re-entrant.
 drupal_ti_ensure_drupal
 
+# Download module dependencies.
+mkdir -p "$DRUPAL_TI_DRUPAL_DIR/$DRUPAL_TI_MODULES_PATH"
+cd "$DRUPAL_TI_DRUPAL_DIR/$DRUPAL_TI_MODULES_PATH"
+git clone --branch 8.x-1.x http://git.drupal.org/project/composer_manager.git --depth 1
+git clone --branch 8.x-1.x http://git.drupal.org/project/profile.git --depth 1
+git clone --branch 8.x-1.x http://git.drupal.org/project/address.git --depth 1
+git clone --branch 8.x-1.x http://git.drupal.org/project/entity.git --depth 1
```
Comment (andrewbelcher) [inline travis-ci/travis-before-script.sh]: Can we use composer to download these rather than manually cloning them?

**Comment by andrewbelcher** [comment]
Does the feature need to live under crm? It could live straight under decoupled_auth as it's usable  outside crm and that makes the directory much nicer :-P

**Comment by yanniboi** [comment]
Now using composer to get module dependencies.

**Pull request merged**